pom.xml

```diff
@@ -175,6 +177,36 @@
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
```

Comment:
I suspect it should be in `<plugins>` and not in `pluginManagement` to run in every module?

Reply:
Moved